docs/compatibility.md

```diff
@@ -366,9 +366,10 @@ similar issue exists for writing dates as described
 to work for dates after the epoch as described
 [here](https://github.com/NVIDIA/spark-rapids/issues/140).
 
-The plugin supports reading `uncompressed`, `snappy` and `zlib` ORC files and writing `uncompressed`
- and `snappy` ORC files.  At this point, the plugin does not have the ability to fall back to the
- CPU when reading an unsupported compression format, and will error out in that case.
+The plugin supports reading `uncompressed`, `snappy`, `zlib` and `zstd` ORC files and writing
+ `uncompressed`, `snappy` and `zstd` ORC files.  At this point, the plugin does not have the ability
```

Comment:
I didn't realize writing was supported but I see the cudf issue is resolved to support it, but I don't see that we have any tests for it.  Perhaps we should file an issue to enable tests?  Also note we still have https://github.com/NVIDIA/spark-rapids/issues/3037 open so we can probably close it.

Reply:
I guess this is covered by https://github.com/NVIDIA/spark-rapids/pull/6362/files

Reply:
Agreed, there are some unanswered questions relating to output file sizes when using the ZSTD compression support, as the zstd block size used by the GPU doesn't match the default block size used by the CPU when writing Parquet+zstd.

Reply:
Would it be better to document that writing zstd for Parquet and ORC is experimental, and can be enabled  by setting LIBCUDF_NVCOMP_POLICY=ALWAYS ?  

Setting to draft until 6362 is merged.

Reply:
personally I would almost just do the read support in this PR and the write support should be documented with the other PR.

Reply:
Agree with @tgravescs, I see no reason to speculate in this PR when we can (and should) update the docs in the PR that adds the functionality in question.  Only reason we're adding docs in this PR is that the doc updates were missed when ZSTD read support tests went in via #5898.

Reply:
Updated to only indicate zstd read support.